client/src/urls.js

```diff
@@ -1,4 +1,4 @@
-const add_definition_url = "https://docs.google.com/forms/d/e/1FAIpQLSfKF0yyleI5XdPVtl-bEuQUGy2HZPfnUU-e2sDjL31eLuygUA/viewform?usp=sf_link";
+const add_definition_url = "/defineform";
```

Comment:
I want to leave this change for later – I think it would be good to do a sort of "soft release" where we push the new form to prod and test it out before switching the links over.

Reply:
Makes sense.